orchagent/portsorch.cpp

```diff
@@ -352,7 +353,8 @@ static bool isValidPortTypeForLagMember(const Port& port)
  *    bridge. By design, SONiC switch starts with all bridge ports removed from
  *    default VLAN and all ports removed from .1Q bridge.
  */
-PortsOrch::PortsOrch(DBConnector *db, DBConnector *stateDb, vector<table_name_with_pri_t> &tableNames, DBConnector *chassisAppDb) :
+PortsOrch::PortsOrch(DBConnector *db, DBConnector *stateDb, vector<table_name_with_pri_t> &tableNames,
```

Comment:
Can you please revert this change if its not modified?

Reply:
Done